concrete/single_pages/dashboard/system/basics/editor.php

```diff
@@ -19,59 +19,70 @@
 }
 </style>
 <form id="ccm-editor-config" method="post" class="ccm-dashboard-content-form" action="<?= $view->action('submit') ?>">
-    <?php $token->output('submit') ?>
+<?php $token->output('submit') ?>
     <div class="row">
         <div class="col-md-6">
-            <?= $form->label('', t('concrete5 Extensions')) ?>
-            <div class="checkbox">
-                <label>
-                    <?= $form->checkbox('enable_filemanager', 1, $filemanager) ?>
-                    <?= t('Enable file selection from file manager.') ?>
-                </label>
-            </div>
-            <div class="checkbox">
-                <label>
-                    <?= $form->checkbox('enable_sitemap', 1, $sitemap) ?>
-                    <?= t('Enable page selection from sitemap.') ?>
-                </label>
-            </div>
-            <?= $form->label('', t('Editor Plugins')) ?>
-            <?php
-            foreach ($plugins as $key => $plugin) {
-                if (!in_array($key, $selected_hidden)) {
-                    $description = $plugin->getDescription();
-                    ?>
-                    <div class="checkbox">
+            <fieldset>
+                <legend><?= t('concrete5 Extensions') ?></legend>
+                <div class="form-group">
+                    <div class="form-check">
                         <label>
```

Comment:
make sure to use form-check-label and proper bootstrap 4 classes here.

Reply:
Done